src/applications/ivc-champva/shared/utilities.js

```diff
@@ -4,31 +4,28 @@ export function applicantWording(
   formData,
   context,
   isPosessive = true,
-  cap = true,
-  index,
+  _cap = true, // This doesn't matter now that we don't use 'you'/'your'
+  _index, // Will be unused now that we don't use 'you'/'your'
```

Comment:
Leaving these arguments in for the time being incase we want to reinstate this functionality.